test-data/unit/check-overloading.test

```diff
@@ -787,7 +787,11 @@ class D(A):
     def f(self, x: str) -> str: return ''
 [out]
 tmp/foo.pyi:12: error: Signature of "f" incompatible with supertype "A"
+tmp/foo.pyi:12: note: Signature of "f" in superclass: "Overload(def f(self, x: builtins.int) -> builtins.int, def f(self, x: builtins.str) -> builtins.str)"
```

Comment:
@JukkaL it would probably be difficult/messy to split the output between multiple `note` outputs.  Would it suffice to have one `note` output multiple lines?